django/db/models/lookups.py

```diff
@@ -40,6 +40,9 @@ def apply_bilateral_transforms(self, value):
             value = transform(value)
         return value
 
+    def __repr__(self):
+        return f'{self.__class__.__name__}(lhs={self.lhs!r}, rhs={self.rhs!r})'
```

Comment:
Not convinced kwargs notation adds any value to the representation, I'd personally drop them

```suggestion
        return f'{self.__class__.__name__}({self.lhs!r}, {self.rhs!r})'
```